src/constants/inputKeys.ts

```diff
@@ -3,6 +3,7 @@ const INPUT_KEYS = Object.freeze({
   ASSIGNEE_REQUIRED: 'assigneeRequired',
   CHECKLIST_REQUIRED: 'checklistRequired',
   SEMANTIC_TITLE_REQUIRED: 'semanticTitleRequired',
+  SEMANTIC_PR_TITLE_REQUIRED: 'semanticPRTitleRequired',
```

Comment:
bunu branch name kontrolü için mi koyduk, pr title'ı için mi koyduk

SEMANTIC_BRANCH_NAME_REQUIRED
SEMANTIC_PR_TITLE_REQUIRED
  
 böyle ilerletsek daha iyi değil mi